core/src/main/java/novoda/lib/sqliteprovider/util/UriUtils.java

```diff
@@ -6,29 +6,28 @@
 
 public class UriUtils {
 
-    private static Map<String, String> mappedIds = new HashMap<String, String>();
+    private UriUtils() throws IllegalAccessException {
+        throw new IllegalAccessException("Utils class shouldn't be instantiated");
```

Comment:
My train of thought was that someone might try to instantiate the class reflectively (don't ask me why). But I agree, simply not doing anything would be cleaner 😁